kong/plugins/syslog/schema.lua

```diff
@@ -6,6 +6,16 @@ local severity = {
   one_of = { "debug", "info", "notice", "warning", "err", "crit", "alert", "emerg" },
 }
 
+local facility = {
+  type = "string",
+  default = "USER",
+  one_of = { "AUTH", "AUTHPRIV",  "CRON",  "DAEMON",
+  "FTP",  "KERN",  "LPR",  "MAIL",
+  "NEWS", "SYSLOG",  "USER",  "UUCP",
+  "LOCAL0",  "LOCAL1", "LOCAL2",  "LOCAL3",
+  "LOCAL4",  "LOCAL5",  "LOCAL6",  "LOCAL7" },
```

Comment:
```suggestion
  one_of = { "AUTH", "AUTHPRIV", "CRON", "DAEMON",
             "FTP", "KERN", "LPR", "MAIL",
             "NEWS", "SYSLOG", "USER", "UUCP",
             "LOCAL0", "LOCAL1", "LOCAL2", "LOCAL3",
             "LOCAL4", "LOCAL5", "LOCAL6", "LOCAL7" },
```
style; single spaces, and alignment